### ISSUE TITLE
Home page has no headings

### ISSUE THREAD

**Issue by fstorr** [hackoregon/civic#822]
## What

Headings `<h1>` through to `<h6>` are important navigation aids for assistive technology users. They're also great for your SEO. The home page has headings (Education, Transportation, etc.) but they're not marked up as such. Headings should be nested correctly: the page's title should be an `<h1>` (in this case "Making Public Information Public Knowledge" would be sensible) and then sub-headings from that would be `<h2>`s.

See attached annotated screen shot for heading structure.

![annotated screen shot of home page with preferred heading structure](https://user-images.githubusercontent.com/169471/63480323-496ac680-c446-11e9-9e6f-388c2cedfef9.png)

**Comment by Dianna** [comment]
Closing because homepage redesign in progress
